rviz_tactile_plugins/src/tactile_contact_display.cpp

```diff
@@ -235,21 +240,18 @@ void TactileContactDisplay::update(float wall_dt, float ros_dt)
 
   Display::update(wall_dt, ros_dt);
 
-  ros::Time now = ros::Time::now();
-  ros::Duration timeout(timeout_property_->getFloat());
+  ros::Time deadline = ros::Time::now() - ros::Duration(timeout_property_->getFloat());
```

Comment:
I think this line is the cause of a crash when loading the plugin, and rosparam use_sim_time = true and no clock is published yet
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  Time is out of dual 32-bit range
```

previous to this fix the plugin did not crash

```
#0  0x00007ffff60a9f47 in __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff60ab8b1 in __GI_abort () at abort.c:79
#2  0x00007ffff6700957 in  () at /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff6706ae6 in  () at /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff6706b21 in  () at /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff6706d54 in  () at /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff3a77f8c in ros::normalizeSecNSecUnsigned(long&, long&) () at /opt/ros/melodic/lib/librostime.so
#7  0x00007ffff3a7d00c in ros::TimeBase<ros::Time, ros::Duration>::operator+(ros::Duration const&) const () at /opt/ros/melodic/lib/librostime.so
#8  0x00007ffff3a7d07f in ros::TimeBase<ros::Time, ros::Duration>::operator-(ros::Duration const&) const () at /opt/ros/melodic/lib/librostime.so
#9  0x00007fff93dc734a in rviz::tactile::TactileContactDisplay::update(float, float) () at /home/.../work/dev/catkin_ws_agni/devel/lib//librviz_tactile_plugins.so
#10 0x00007ffff7a7e156 in rviz::DisplayGroup::update(float, float) (this=0x5555558b5800, wall_dt=3.59348774, ros_dt=0)
    at /media/local/jenkins/jobs/rviz-agni-melodic-devel-ci-deploy-lsp-famula-nightly/workspace/src/rviz/display_group.cpp:240
#11 0x00007ffff7b48595 in rviz::VisualizationManager::onUpdate() (this=0x555556a461e0)
    at /media/local/jenkins/jobs/rviz-agni-melodic-devel-ci-deploy-lsp-famula-nightly/workspace/src/rviz/visualization_manager.cpp:361
#12 0x00007ffff6cb6555 in QMetaObject::activate(QObject*, int, int, void**) () at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
#13 0x00007ffff6cc2fc7 in QTimer::timeout(QTimer::QPrivateSignal) () at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
#14 0x00007ffff6cc3328 in QTimer::timerEvent(QTimerEvent*) () at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
#15 0x00007ffff6cb707b in QObject::event(QEvent*) () at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5

```